etc/ci.sh

```diff
@@ -37,18 +31,62 @@ else
         make
         cd ../..
 
-        sh bin/gap.sh --cover coverage tst/${TEST_SUITE}.g
+        sh bin/gap.sh -q tst/extractmanuals.g
 
-        # generate coverage report
-        sh bin/gap.sh -q <<GAPInput
-            if LoadPackage("profiling") <> true then
-                Print("ERROR: could not load profiling package");
-                FORCE_QUIT_GAP(1);
-            fi;
-            OutputJsonCoverage("coverage", "coverage.json");
+        for ch in tst/testmanuals/*.tst
+        do
+            COVNAME="coverage.`basename $ch .tst`"
+            sh bin/gap.sh -q --cover $COVNAME <<GAPInput
+            Read("tst/testmanuals.g");
+            TestManualChapter("$ch");
             QUIT_GAP(0);
 GAPInput
-    fi
-    cd bin/x86* ; gcov -o . ../../src/*
-    cd ../..
-fi;
+
+            sh bin/gap.sh -q <<GAPInput
+                if LoadPackage("profiling") <> true then
+                    Print("ERROR: could not load profiling package");
+                    FORCE_QUIT_GAP(1);
+                fi;
+                OutputJsonCoverage("$COVNAME", "$COVNAME.json");
+                QUIT_GAP(0);
+GAPInput
+
+        done
+        cd bin/x86* ; gcov -o . ../../src/*
+        cd ../..
+        ;;
```

Comment:
I would suggest moving these two commands after the `esac`. It's always the same -- and it doesn't hurt if we run it for the `makemanuals` target, too does it?